h2o-core/src/main/java/water/parser/PackedDomains.java

```diff
@@ -1,117 +1,131 @@
 package water.parser;
 
+import water.MemoryManager;
 import water.util.StringUtils;
 
-import java.io.*;
-import java.util.Arrays;
-
 import static water.util.ArrayUtils.*;
-/**
- * Recreated "packed domains" functionality, with tests
- * 
- * Created by vpatryshev on 4/12/17.
- */
+
 public class PackedDomains {
+
   public static int sizeOf(byte[] domain) {
     return encodeAsInt(domain, 0);
```

Comment:
I know this is not part of this change, but encode/decodeAsInt are imho duplicate of UnsafeUtils.get/set4.
Also the name seems to be confusing, at least for me, I would expect inverted behavior for decode/encode.